### PR TITLE
FIX: submit_job improvement to allow the possibility to pass a server path

### PR DESCRIPTION
**Pull request by maxcapodi78**
#4662

### CODE REVIEW

File: pyaedt/desktop.py [hunk 1]
```diff
@@ -1816,43 +1825,47 @@ def submit_job(
                 return False
         else:
             if not os.path.exists(aedt_full_exe_path):
-                self.logger.error("AEDT shared path does not exist. Provide a full path.")
-                return False
+                self.logger.warning("The AEDT executable path not visible from the client.")
```
Comment (dcrawforAtAnsys) [inline pyaedt/desktop.py:1828]: Is this client or server? Shouldn't the exe be visible on the server?

**Review by dcrawforAtAnsys — APPROVED** [review]
One minor point regarding the message to be resolved. Otherwise all good with some minor linguistic modifications.

File: pyaedt/desktop.py (hunk 1 above)
Comment (maxcapodi78) [reply pyaedt/desktop.py:1828]: yes that is the server, so the client not necessarly is able to see it. That's why we cannot force the return False if we are not able to see it from the client

**Review by dcrawforAtAnsys — COMMENTED** [review]
LGTM.

**Pull request merged**